### PR TITLE
fix: add some type safety

### PR DESCRIPTION
**Pull request by shaftoe**
Ref: 

- https://github.com/probot/create-probot-app/issues/322
- https://github.com/probot/create-probot-app/issues/315

### CODE REVIEW

File: src/helpers/user-interaction.ts [hunk 1]
```diff
@@ -134,36 +149,46 @@ function getQuestions(
...
-  program: commander.Command,
-  destination: string
-): Promise<Answers> {
+/**
+ * Prompt the user for mandatory options not set via CLI
+ *
+ * @param config Configuration data already set via CLI options
+ *
+ * @returns
+ * merged configuration options from CLI and user prompt
```
Comment (gr2m) [inline src/helpers/user-interaction.ts:158]: it looks a bit odd  that the description is in the next line. Is that intended?

**Review by gr2m — APPROVED** [review]
Looks great, just a question about the `@return` JSDoc statements

File: src/helpers/user-interaction.ts (hunk 1 above)
Comment (shaftoe) [reply src/helpers/user-interaction.ts:158]: I think it's cargo-cult motivated 😅 just saw it done that way somewhere else probably. Let me put them on same line

Comment (shaftoe) [reply src/helpers/user-interaction.ts:158]: Found the source: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/inquirer/index.d.ts

Comment (gr2m) [reply src/helpers/user-interaction.ts]: thanks! Good to know you can do this :)

Comment (shaftoe) [reply src/helpers/user-interaction.ts]: Do you mean adding types or going full cargo cult? :)
Jokes aside, I'm taking the chance to improve my TS skills so if you see anything odd please let me know, I won't be offended but quite the opposite, right now is still "fake it till you make it"

**Pull request merged**